src/components/Tutorial/TutorialNavigationButton.tsx

```diff
@@ -14,7 +14,6 @@ const TutorialNavigationButton = React.forwardRef<
 >(({ clickHandler, value, disabled = false, classes }, ref) => (
   <a
     className={cx(anchorButtonRecipe({ variableWidth: true, smallGapX: true }), classes)}
-    onClick={clickHandler}
```

Comment:
it's duplicated by `fastClick`, which caused a skip of step I've painfully experienced multiple times while testing

Reply:
Yes, good catch. It doesn't make sense to have both.